spec/models/post_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper' # Return character is CR+LF on Windows.
+
+RSpec.describe Post, type: :model do
+  describe 'Post validations' do
+    user1 = User.create(name: 'Lolade', bio: 'Public Administator', posts_counter: 0)
```

Comment:
- [ ] Unfortunately, now I am getting this error when trying to run your tests :confused: 
![image](https://user-images.githubusercontent.com/62705947/190634865-09d13e4f-a026-427d-b06a-002244b64326.png)